.eslintrc.json

```diff
@@ -1,4 +1,7 @@
 {
+  "env": {
+    "es6": true
```

Comment:
This PR doesn't seem like the right place for this change. If this isn't going into 1.2 anyway, it could perhaps wait until after we decide how to handle ES6+ syntax, or it could go in sooner using function prototypes instead of classes.

Reply:
I think this is not the right place to do this @svinkle can you remove this commit and open a different PR to discuss it?

Reply:
@smhigley I have changed the tabbed carousel example to use prototoype, instead of class, so should not have linting problems with the current tests